watchdog/watchdog.go

```diff
@@ -22,12 +22,36 @@ const (
 )
 
 // Program is a program at path with arguments
-type Program struct {
+type ProgramNormal struct {
 	Path   string
 	Args   []string
 	ExitOn ExitOn
 }
 
+func (p *ProgramNormal) GetPath() string {
```

Comment:
If these are getters, you don't want pointer... e.g. `func (p program) GetPath() string {`